### PR TITLE
WIP: initial macos ansible

### PR DESCRIPTION
**Pull request by gdams**
closes: https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/212

### CODE REVIEW

File: ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Ant-Contrib/tasks/main.yml [hunk 1]
```diff
@@ -15,7 +15,16 @@
...
     timeout: 25
     validate_certs: no
-  when: antcontrib_status.stat.exists == False
+  when:
+    - antcontrib_status.stat.exists == False
+    - ansible_os_family != "Darwin"
+  tags: ant-contrib
+
+- name: Download ant-contrib macOS
+  command: "/usr/local/bin/wget --no-check-certificate -o /tmp/ant-contrib-1.0b2-bin.tar.gz https://sourceforge.net/projects/ant-contrib/files/ant-contrib/ant-contrib-1.0b2/ant-contrib-1.0b2-bin.tar.gz"
```
Comment (bblondin) [inline ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Ant-Contrib/tasks/main.yml]: why not use ```get_url```


```
- name: Download Download ant-contrib macOS
  get_url:
    url: https://sourceforge.net/projects/ant-contrib/files/ant-contrib/ant-contrib-1.0b2/ant-contrib-1.0b2-bin.tar.gz
    dest: /tmp/ant-contrib-1.0b2-bin.tar.gz 
    mode: 0440
    validate_certs: no

```

Comment (gdams) [reply ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Ant-Contrib/tasks/main.yml]: I'm having issues using the get_url module on macos, investigating now

**Comment by gdams** [comment]
I can change the java stuff to install our own homebrew repo now: https://github.com/AdoptOpenJDK/homebrew-openjdk

**Comment by karianna** [comment]
@gdams This ready for review now or still WIP?

**Comment by sxa555** [comment]
ping @gdams - is this in a state where it's got a chance of working, or will likely break everything? ;-)

**Comment by karianna** [comment]
It needs rebasing now as well.

**Pull request closed without merging**